### ISSUE TITLE
Gatsby remarkCreator: image previewSrc not rendering properly

### ISSUE THREAD

**Issue by kendallstrautman** [tinacms/tinacms#434]
## Description

<img width="435" alt="Screen Shot 2019-11-04 at 10 44 14 AM" src="https://user-images.githubusercontent.com/36613477/68134507-38cbd780-fef0-11e9-80fa-d482aba6f9fc.png">

If you add an image field to the remarkCreatorButton, for let's say a hero image, if you pass the path to image (e.g. `/content/images/filename.jpg`) and your images don't live in static, the preview image doesn't render bc of the way gatsby handles images. 

I've thought about trying to get ahold of this image with a query structure similar to this:

``` graphql
query MyQuery {
  imageSharp(fluid: {originalName: {eq: "some-filename.jpg"}}) {
    fluid {
      src
      originalName
    }
  }
}
```
This won't work for the BlogList component because it is not a page component, so we would have to use `StaticQuery` or `useStaticQuery`. Unfortunately its not currently possible to pass in parameters to `StaticQuery` and we would need that `originalName` filter to be dynamic.

I've done a little research and ran into some issues that are related. 
- https://github.com/gatsbyjs/gatsby/issues/5244
- https://github.com/gatsbyjs/gatsby/issues/2293
- https://github.com/gatsbyjs/gatsby/issues/10482 ✨

TL;DR --> As I understand it, the biggest issue is that in order for this to work we would need to be able to pass in a dynamic argument (the filename) to a static query.

### Steps to reproduce

Get a clone of [brevifolia-gatsby-tinacms](https://github.com/kendallstrautman/brevifolia-gatsby-tinacms) going
Start up the dev env - `yarn dev`
Open the sidebar on the home page and add a new blog with an image
Head to the [blog list component](https://github.com/kendallstrautman/brevifolia-gatsby-tinacms/blob/master/src/components/BlogList.js) to see the file where this is handled. 

### Expected result

The preview source would render

### Actual result

See initial image

### Environment
MacOS
```
"gatsby": "^2.13.9",
 "gatsby-image": "^2.2.8",
"gatsby-plugin-sharp": "^2.2.3",
 "gatsby-plugin-tinacms": "^0.1.8",
 "gatsby-remark-images": "^3.1.3",
```

**Comment by kendallstrautman** [comment]
We're assuming this isn't possible with `Gatsby` for now. If anyone finds a workaround, please post!
